### ISSUE TITLE
[Cisco Secure Email Gateway] Supporting AsyncOS v15

### ISSUE THREAD

**Issue by jamiehynds** [elastic/integrations#10979]
Our Cisco Secure Email Gateway was built and tested against AsyncOS v14, however v15 is now available and we need to ensure we support this latest release. Release notes for v15 are available here: https://www.cisco.com/c/dam/en/us/td/docs/security/esa/esa15-0/release_notes/Secure_Email_15-0_Release_Notes.pdf

Can we please test our integration against v15 and ensure support is reflected in our docs accordingly.

**Comment by qcorporation** [comment]
@jamiehynds @cpascale43, please let us know when you have logs from v15 - happy to help with testing the integration and patching issues if there are gaps in parsing the logs

**Comment by jamiehynds** [comment]
@qcorporation Cisco have granted us licenses and appliance download for their Email Gateway. Ok to add this appliance to our list and generate v15 logs from there?

**Comment by qcorporation** [comment]
@jamiehynds yes, I didn't put two and two together. We are starting to explore log extraction happy to add this to the list of appliances to get from Cisco
